sql/moz-fx-data-shared-prod/contextual_services_derived/suggest_revenue_levers_daily_v1/query.sql

```diff
@@ -27,7 +27,7 @@ WITH client_shares AS (
       END
     ) AS eligible_clients
   FROM
-    telemetry.unified_metrics
+    `moz-fx-data-shared-prod.telemetry.unified_metrics`
```

Comment:
@alekhyamoz  There are some `contextual_services` revenue datasets which got fully qualified in this PR and need your approval. (Or we can put them on a skip list if references should for some reason not be fully-qualified)